internal/reminder/check_snoozed_until.go

```diff
@@ -15,8 +15,6 @@ func hasSnoozedUntil(ctx context.Context, logger log.Logger, incident model.Inci
 			log.Trace(),
 			log.Action("do_not_notify"),
 			log.Reason("isPaused"),
-			log.NewValue("channelID", incident.ChannelId),
```

Comment:
Suggestion: rename the parameter logger to logWriter to make it clear that it already contains the log fields.